lib/policies/index.js

```diff
@@ -27,7 +28,12 @@ class Policies {
     policy.policy = (params, ...args) => {
       const validationResult = validate(params);
       if (validationResult.isValid) {
-        return action(params, ...args);
+        const act = action(params, ...args);
+        return (req, res, next) => {
+          bus.emit('EG::policy::beforePolicy', params, req, res, next);
+          act(req, res, next);
+          bus.emit('EG::policy::afterPolicy', params, req, res, next);
```

Comment:
this part will be unstable. as mentioned in comments the moment you policy does any async call and return act will be released and bus.emit after will fire. 

so timming will be 
EG::policy::beforePolicy
some async action start
EG::policy::afterPolicy
async action finish
res.send

why do you need after event? What if I suggest to just leave before and remove after?

Reply:
@DrMegavolt would be fine to just leave before event. Thank you